### PR TITLE
RK-19823 - add new DOP params

### PR DESCRIPTION
**Pull request by DavidRBiderman**

### CODE REVIEW

**Comment by DavidRBiderman** [comment]
Github Enforcer opened Task: [RK-19823](https://rookout.atlassian.net/browse/RK-19823)

[RK-19823]: https://rookout.atlassian.net/browse/RK-19823?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

File: charts/rookout-hybrid/values.yaml [hunk 1]
```diff
@@ -163,6 +163,9 @@ datastore:
    # -- whether to deploy a Datastore
   enabled: true
 
+   # -- whether to enable the new DAO wrappers for the datastore
+  enableNewDao: false
```
Comment (Urook) [inline charts/rookout-hybrid/values.yaml:167]: You need to add it to the readme

**Review by Urook — CHANGES_REQUESTED** [review]

**Comment by gosharo** [comment]
@DavidRBiderman why do you change these charts? it is customer facing

**Review by gosharo — CHANGES_REQUESTED** [review]
@DavidRBiderman why do you change these chart? It is customer-facing and deprecated
https://github.com/Rookout/helm-charts#available-charts

**Comment by DavidRBiderman** [comment]
> @DavidRBiderman why do you change these chart? It is customer-facing and deprecated https://github.com/Rookout/helm-charts#available-charts
@gosharo 
@nissim-infra asked to add it to the legacy chart in case someone still uses it and they upgrade the datastore

File: charts/rookout-hybrid/README.md [hunk 2]
```diff
@@ -125,6 +125,7 @@ helm upgrade --install rookout rookout/rookout-hybrid \
 | datastore.serviceAccount.create | bool | `true` | Specifies whether a service account should be created |
 | datastore.serviceAccount.name | string | `""` | The name of the service account to use. If not set and create is true, a name is generated using the fullname template |
 | datastore.tolerations | list | `[]` | [Tolerations] for use with node taints |
+| datastore.enableNewDao | bool | `false` | Enable new SQL connector and new DAO schema
```
Comment (Urook) [inline charts/rookout-hybrid/README.md]: Should we not also recommend increasing the cpu in that case?

Comment (DavidRBiderman) [reply charts/rookout-hybrid/README.md]: @ElDuderinos  received all the disclaimers. It does not require nothing but it is recommended that the client increase resources generally speaking. This is not a mandatory step

Comment (Urook) [inline charts/rookout-hybrid/README.md]: @gosharo Should we bump to 0.3.2 (at the top)?

**Review by Urook — CHANGES_REQUESTED** [review]

**Comment by gosharo** [comment]
as @DavidRBiderman asked me to tag,
@dudiRookout @DavidRBiderman  this is should be documented in docs.rookout.com.
Not relevent to charts.. it's applciation level configuration and not infra level.

**Comment by gosharo** [comment]
as @DavidRBiderman asked me to tag,
@dudiRookout @DavidRBiderman this is should be documented in docs.rookout.com.
Not relevent to charts.. it's applciation level configuration and not infra level.

**Pull request closed without merging**